frontend/src/components/Header/Header.tsx

```diff
@@ -74,28 +75,53 @@ function InstallationPicker(page: string) {
     const { TranslateText } = useLanguageContext()
     const [allPlantsMap, setAllPlantsMap] = useState<Map<string, string>>()
     const { installationCode, switchInstallation } = useInstallationContext()
+    const [showActivePlants, setShowActivePlants] = useState<boolean>(true)
+    const [isDialogOpen, setIsDialogOpen] = useState<boolean>(false)
+    const [updateListOfActivePlants, setUpdateListOfActivePlants] = useState<boolean>(false)
+
     useEffect(() => {
-        BackendAPICaller.getEchoPlantInfo().then((response: EchoPlantInfo[]) => {
+        const plantPromise = showActivePlants ? BackendAPICaller.getActivePlants() : BackendAPICaller.getEchoPlantInfo()
+        plantPromise.then(async (response: EchoPlantInfo[]) => {
             const mapping = mapInstallationCodeToName(response)
             setAllPlantsMap(mapping)
         })
-    }, [])
+    }, [showActivePlants, updateListOfActivePlants])
+
     const mappedOptions = allPlantsMap ? allPlantsMap : new Map<string, string>()
     return (
-        <Autocomplete
-            options={Array.from(mappedOptions.keys()).sort()}
-            label=""
-            disabled={page === 'mission'}
-            initialSelectedOptions={[installationCode.toUpperCase()]}
-            placeholder={TranslateText('Select installation')}
-            onOptionsChange={({ selectedItems }) => {
-                const mapKey = mappedOptions.get(selectedItems[0])
-                if (mapKey !== undefined) switchInstallation(mapKey)
-                else switchInstallation('')
-            }}
-            autoWidth={true}
-            onFocus={(e) => e.preventDefault()}
-        />
+        <>
+            <Autocomplete
+                options={Array.from(mappedOptions.keys()).sort()}
+                label=""
+                disabled={page === 'mission'}
+                initialSelectedOptions={[installationCode.toUpperCase()]}
+                placeholder={TranslateText('Select installation')}
+                onOptionsChange={({ selectedItems }) => {
+                    const mapKey = mappedOptions.get(selectedItems[0])
+                    if (mapKey !== undefined) switchInstallation(mapKey)
+                    else switchInstallation('')
+                }}
+                autoWidth={true}
+                onFocus={(e) => {
+                    e.preventDefault()
+                    setUpdateListOfActivePlants(!updateListOfActivePlants)
+                }}
+            />
+            <Icon name={Icons.Settings} size={16} aria-haspopup="dialog" onClick={() => setIsDialogOpen(true)} />
```

Comment:
Icon size maybe change to 24 so that it's a bit bigger

Reply:
Also maybe change the color to the other icons as well 

            <Icon name={Icons.Settings} size={24} aria-haspopup="dialog" color={tokens.colors.interactive.primary__resting.rgba} onClick={() => setIsDialogOpen(true)} />

Reply:
Also, so that we keep the same format as the other buttons in the heather, i suggest changing the code to this:

<Button variant="ghost_icon" onClick={() => setIsDialogOpen(true)} >
                <Icon name={Icons.Settings} size={24} aria-haspopup="dialog" color={tokens.colors.interactive.primary__resting.rgba} />
            </Button>
            
 For consistency and because it's more clear when you hover and see it highlighted

Reply:
I would update all the icons in this file to 24 but could be because I'm also using aa small screen, you can decide on that